roles/openshift_prometheus/tasks/main.yaml

```diff
@@ -1,2 +1,37 @@
 ---
 # tasks file for openshift_prometheus
+
+#- name: Gather OpenShift Prometheus Facts
+#  openshift_prometheus_facts:
+#    oc_bin: "{{openshift.common.client_binary}}"
+#    openshift_prometheus_namespace: "{{openshift_prometheus_namespace}}"
+
+- name: Add prometheus project
+  oc_project:
+    state: present
+    name: "{{ openshift_prometheus_namespace }}"
+    description: Prometheus
+
+- name: generate prometheus deployment
+  template:
+    src: prometheus.j2
+    dest: "{{ prometheus_template_path }}"
```

Comment:
I do not know if this is required, but in logging we generate the deployments to a tmp folder generated everytime and cleaned up after ansible runs.